tools/clang/test/HLSLFileCheck/d3dreflect/mesh-groupshared.hlsl

```diff
@@ -1,11 +1,11 @@
-// RUN: %dxilver 1.8 | %dxc -T lib_6_7 %s | %D3DReflect %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -T lib_6_7 -Vd -validator-version 0.0 %s | %D3DReflect %s | FileCheck %s
```

Comment:
Should we have another RUN line test validator 1.7?

Reply:
No, this is data that is only in the additional RDAT beyond 1.8.  The whole purpose of this test is to test that data, so the test requires this validator version to allow the data to be set in the RDAT.  Same story with other ones I added this to.